src/test/java/org/apache/maven/shared/utils/io/FileUtilsTest.java

```diff
@@ -438,9 +448,300 @@ public void copyFile1()
             testFile1.lastModified() == destination.lastModified());*/
     }
 
+    private static long MILLIS_PER_DAY = 24 * 60 * 60 * 1000L;
+
+    /** A time today, rounded down to the previous minute */
+    private static long MODIFIED_TODAY = (System.currentTimeMillis() / 60_000) * 60_000;
```

Comment:
Throwing away seconds here allows these precise times to be used accurately even on low resolution file systems such as FAT32.